src/engine/keycontrol.cpp

```diff
@@ -287,18 +287,26 @@ void KeyControl::slotPitchChanged(double pitch) {
 
 void KeyControl::updatePitch() {
     double pitch = m_pPitch->get();
+    double pitchKnobRatio = KeyUtils::semitoneChangeToPowerOf2(pitch);
```

Comment:
done -- pitchknobratio is used inside and outside the if so it can't be a lot closer